BP/common/religion/religions/00_christianity.txt

```diff
@@ -522,14 +519,13 @@
 			color = { 0.8 0.3 0.5 }
 			religious_head = d_patriarchate_in_the_east
 
-			#Smarkand was part of the furthest east province, but Merv was the Metropilan See
-			#Using Samarkand for now
+			#WtWSMS: alexandria, antioch, baghdad, jerusalem, rayy
+			#Move holy sites East to avoid dependency on Roman Empire sites (as per history)
```

Comment:
Doesn't make sense to railroad that, the eastern expansion happens later. In 476 the centre of gravity is still very much Persia and the Roman Empire, certainly not Samarkand or Kerala, so Alexandria, Antioch and Jerusalem are much more important.

Reply:
On the other hand, the Church of the East became the national church of the Persian Empire in 410 in the [Council of Seleucia-Ctesiphon](https://en.wikipedia.org/wiki/Council_of_Seleucia-Ctesiphon) and asserted independence from Rome in 424. So the split was fairly solidified by the 470s. Maybe Antioch should remain, but I'm not certain about Alexandria. 

As for Kerala, that's because the Church of the East had suzerainty over the St. Thomas Christians/Nasrani who were most definitely present (in small numbers anyway) by 476.

It's unfortunate we can't dynamically modify holy sites as that would resolve this issue; but I'll change it to antioch/alexandria.

Reply:
I'm not saying there was no significant rift, I'm saying that with regards to Nestorianism it had not yet switched the focus to the East as it would do later. After all, there can be a solid split sharing the same important sites.